src/core/event/group.h

```diff
@@ -57,9 +58,9 @@ class xlio_group {
     void add_ring(ring *);
     void del_ring(ring *);
 
-    inline unsigned get_flags() { return m_flags; }
-    inline event_handler_manager_local *get_event_handler() { return m_event_handler; }
-    inline tcp_timers_collection *get_tcp_timers() { return m_tcp_timers; }
+    inline unsigned get_flags() const { return m_flags; }
```

Comment:
AFAIK inline is not needed when implemented in place. This is implicit.

Reply:
will remove

Reply:
Done.